error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/PrimitiveComparisonTest.java

```diff
@@ -12,6 +12,8 @@
 // XXX: Can we perhaps work-around this by describing the fixes in reverse order?
 final class PrimitiveComparisonTest {
   /**
+   * Identification test for primitive {@code byte} comparisons.
```

Comment:
Proposal to stick with the usual grammatical form:
```suggestion
   * Tests the identification of {@code byte} comparisons for which boxing can be avoided.
```